modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,65 @@
+[id="gitops-logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+By logging in to the Keycloak console after configuring the Argo CD instance, you can manage identities or roles and define the permissions assigned to the various roles.
```

Comment:
configuring the Argo CD instance must be mentioned as prereq as suggested below. Suggest rewording this to: Log in to the Keycloak console to manage identities or roles and define the permissions assigned to the various roles